src/v/cluster/cluster_discovery.h

```diff
@@ -81,28 +81,28 @@ class cluster_discovery {
     // assignment of node IDs for the seeds.
     ss::future<model::node_id> determine_node_id();
 
-    /**
-     * Returns brokers to be used to form a Raft group for a new cluster.
-     *
-     * If this node is a cluster founder, returns all seed servers, assuming
-     * all founders are configured with identical seed servers list.
-     * If this node is not a cluster founder, returns an empty list.
-     * In case of Emtpy Seed Cluster Bootstrap, that reflects to a list of the
-     * root broker in root if cluster is not there yet, and empty otherwise.
-     * \throw std::runtime_error if cluster needs a bootstrap but seed brokers
-     * do not satisfy relevant validation
-     */
```

Comment:
I don't think so -- I tend to prefer `//` commenting because it encourages inline comments more than `/**` (fewer lines, less intrusive IMO), and for consistency I just use them everywhere.

Might be worth bringing up more broadly though. I think our style guide isn't fully baked.